mutation/types.ts

```diff
@@ -38,7 +38,8 @@ export type SWRMutationConfiguration<
     key: string,
     config: Readonly<
       SWRMutationConfiguration<Data, Error, ExtraArg, SWRMutationKey, SWRData>
-    >
+    >,
+    arg: any
```

Comment:
@benevbright how about using the ExtraArg type instead of the any type?

Reply:
sorry for late response. I found that my PR is duplicated: https://github.com/vercel/swr/pull/2286
will close this one.